playground/vitest-setup.ts

```diff
@@ -99,6 +99,7 @@ beforeAll(async (s) => {
 	browser = await chromium.connect(wsEndpoint);
 	page = await browser.newPage();
 
+	// @ts-ignore
```

Comment:
This is needed because unfortunately this file is getting pulled in from the various playground apps (since they import from `../../__test-utils__` which imports from this file) and they error saying:
```
error TS2339: Property 'console' does not exist on type 'typeof globalThis'.
```

This seems to be caused by the use of the workers types... I am not really 100% sure what the problem is, I'm trying to figure that out 😕

Reply:
This seemed to do the trick: https://github.com/flarelabs-net/vite-plugin-cloudflare/pull/44/commits/8013a3c8b5d4c5eec56dea85fcf02c87822aebe0 🤷

Reply:
I think we're missing a definition for `globalThis` in the workers types? 😕